.editorconfig

```diff
@@ -1,2 +1,12 @@
-[*.{md,markdown}]
-trim_trailing_whitespace = false
```

Comment:
Actually, this mess up with my IDE by removing the trailing whitespace on the markdown documents. Which impact the display of the .md

Reply:
I get it, i left the statement https://github.com/dashevo/wallet-lib/pull/81/files#diff-1e70daafb475c0ce3fef7d2728279182R12